src/problem_depot/problems/socp.jl

```diff
@@ -267,6 +267,18 @@ end
         @test evaluate(H) ≈ Hval atol=atol rtol=rtol
     end
 
+    # https://github.com/jump-dev/Convex.jl/pull/444
+    x = Variable(3)
+    a,b,c,d,e,f = rand(6)
+    M = [2 a-b*im c-d*im;
+         a+b*im 2 e-f*im
+         c+d*im e+f*im 2]
+    y = rand(3)
+    p = minimize(quadform(x-y,M); numeric_type = T)
+    if test 
+        @test evaluate(x) ≈ y
+    end
```

Comment:
We need to call `handle_problem!` to actually solve the problem, and should add the tolerances so they can be customized by the caller:
```suggestion
    p = minimize(quadform(x-y,M); numeric_type = T)
    handle_problem!(p)
    if test 
        @test evaluate(x) ≈ y atol=atol rtol=rtol
    end
```